tutorlimesurvey/templates/limesurvey/build/limesurvey/config.php

```diff
@@ -0,0 +1,102 @@
+/*
+THIS FILE IS AUTOGENERATED BY THE LIMESURVEY docker-entrypoint.sh SCRIPT WHEN THE CONTAINER IS STARTED:
+
+https://github.com/adamzammit/limesurvey-docker/blob/master/docker-entrypoint.sh#L70-L74
+
+TO UPDATE IT:
+
+1. Connect to the running container: docker exec -it limesurvey bash
+2. Copy the file /var/www/html/application/config/config.php to the host machine
+3. Compare the two files and copy any changes from the host machine file to this file
+4. Commit the changes to the host machine file to git
+5. Build a new image: tutor images build limesurvey
+6. Push the new image to Docker Hub: tutor images push limesurvey
+7. When starting the new container, make sure the new config.php is used. If not, remove the old
+config.php from the container and restart it.
+*/
+<?php if (!defined('BASEPATH')) {
+    exit('No direct script access allowed');
+}
+$longName = exec("echo \$TZ"); if (!empty($longName)) {date_default_timezone_set($longName);}
+/*
+| -------------------------------------------------------------------
+| DATABASE CONNECTIVITY SETTINGS
+| -------------------------------------------------------------------
+| This file will contain the settings needed to access your database.
+|
+| For complete instructions please consult the 'Database Connection'
+| page of the User Guide.
+|
+| -------------------------------------------------------------------
+| EXPLANATION OF VARIABLES
+| -------------------------------------------------------------------
+|
+|   'connectionString' Hostname, database, port and database type for
+|    the connection. Driver example: mysql. Currently supported:
+|               mysql, pgsql, mssql, sqlite, oci
+|   'username' The username used to connect to the database
+|   'password' The password used to connect to the database
+|   'tablePrefix' You can add an optional prefix, which will be added
+|               to the table name when using the Active Record class
+|
+*/
+return array(
+    'components' => array(
+        'db' => array(
+            'connectionString' =>'mysql:host={{ MYSQL_HOST }};port=3306;dbname={{ LIMESURVEY_DB_NAME }};',
+            'emulatePrepare' => true,
+            'username' =>'{{ LIMESURVEY_DB_USER }}',
+            'password' =>'{{ LIMESURVEY_DB_PASSWORD }}',
+            'charset' => 'utf8mb4',
+'attributes' => array(),
+            'tablePrefix' =>'',
+        ),
+        {%- raw %}
+        'session' => array (
+			'sessionName'=>'LS-VBXZZORFJZCAKGYI',
```

Comment:
Is it reasonable to leave a fixed session name?

Is this something that should be auto generated?

Reply:
I copied this from the docs: https://manual.limesurvey.org/Optional_settings#Allow_usage_of_session_and_Csrf_Validation_in_iFrame. It's something I prefer to leave as is.